web/auth/auth.go

```diff
@@ -316,7 +316,7 @@ func login(c echo.Context) error {
 				successfulAuthentication = true
 			}
 		} else { // Bad login passphrase
-			if err := limits.CheckRateLimit(inst, limits.AuthType); err != nil {
+			if err := limits.CheckRateLimit(inst, limits.AuthType); err != nil && err == limits.ErrRateLimitExceeded {
```

Comment:
You can replace all the `err != nil && err == limits.ErrRateLimitExceeded` by just `err == limits.ErrRateLimitExceeded `